Net/src/NTPServer.cpp

```diff
@@ -0,0 +1,145 @@
+#include "Poco/Net/NTPServer.h"

```

Comment:
naming, opening braces on new line, only english comments please, remove not used commented code

Reply:
done: for NTPServer test I'll do it next week